tests/testthat/helper-tune-package.R

```diff
@@ -83,33 +87,33 @@ redefer_initialize_catalog <- function(test_env) {
 
 if (rlang::is_installed("splines2")) {
 	rec_df <-
-		recipe(mpg ~ ., data = mtcars) %>%
-		step_corr(all_predictors(), threshold = .1) %>%
-		step_spline_natural(disp, deg_free = 5)
+		recipes::recipe(mpg ~ ., data = mtcars) %>%
+		recipes::step_corr(all_predictors(), threshold = .1) %>%
+		recipes::step_spline_natural(disp, deg_free = 5)
 
 	rec_tune_thrsh_df <-
-		recipe(mpg ~ ., data = mtcars) %>%
-		step_corr(all_predictors(), threshold = tune()) %>%
-		step_spline_natural(disp, deg_free = tune("disp_df"))
+		recipes::recipe(mpg ~ ., data = mtcars) %>%
+		recipes::step_corr(all_predictors(), threshold = tune()) %>%
+		recipes::step_spline_natural(disp, deg_free = tune("disp_df"))
 }
 
 
 
-mod_tune_bst <- boost_tree(trees = tune(), min_n = tune(), mode = "regression")
-mod_tune_rf <- rand_forest(min_n = tune(), mode = "regression")
+mod_tune_bst <- parsnip::boost_tree(trees = tune(), min_n = tune(), mode = "regression")
+mod_tune_rf <- parsnip::rand_forest(min_n = tune(), mode = "regression")
 
 if (rlang::is_installed("probably")) {
 
 	adjust_tune_min <-
```

Comment:
Given that we usually use `rec` in the name of recipes objects, I would like to advocate for calling tailor objects something with `tailor` rather than `adjust_`.